Sources/GDPR/ViewModels/Status.swift

```diff
@@ -18,35 +10,35 @@ enum TermState: Codable {
     case accepted(at: Date)
     case rejected
     case undefined
-    
+
     enum CodingKeys: String, CodingKey {
         case accepted
         case rejected
         case undefined
     }
-    
+
     init(from decoder: Decoder) throws {
         let values = try decoder.container(keyedBy: CodingKeys.self)
         if let value = try? values.decode(Date.self, forKey: .accepted) {
             self = .accepted(at: value)
-        } else if  (try? values.decode(String.self, forKey: .rejected)) != nil {
+        } else if (try? values.decode(String.self, forKey: .rejected)) != nil {
             self = .rejected
         } else if (try? values.decode(String.self, forKey: .undefined)) != nil {
```

Comment:
`} else if (try? values.decode(String.self, forKey: .undefined)) != nil {`
This case is not needed. Will be set to `.undefined` anyway.